### DISCUSSION TITLE
Helics and Gridlab-D interface example run

### DISCUSSION THREAD

**Discussion by H-bhandari** [GMLC-TDC/HELICS#2318] [Q&A]
What does this error mean?
error [2001-01-01 00:00:00 pst] : exec halted: synchronization failed fatal [2001-01-01 00:00:00 pst] : shutdown after simulation stopped prematurely error [2001-01-01 00:00:00 pst] : saveall: unable to open stream 'gridlabd.xml' for writing error [2001-01-01 00:00:00 pst] : dump to 'gridlabd.xml' failed fatal [2001-01-01 00:00:00 pst] : environment startup failed: permission denied

**Comment by phlptp** [comment]
@afisher1 this looks like a gridlabd error,  Maybe you can shed some light on what potential causes might be?

**Comment by afisher1** [comment]
Those are indeed gridlab-d errors. unfortunately that isn't enough output to tell me what's going on. sometimes there are error messages output before this. Were you running your own model or an existing tutorial model. or if you could send me all the output from the GridLAB-D simulator that would also help to trace where your issue is.

**Comment by trevorhardy** [comment]
@H-bhandari, you need any more help on this or should we close it out?
